app/views/user/swaps/_swap_profile_inner.html.haml

```diff
@@ -23,5 +23,5 @@
 - if !hide_polls? && !other_user.constituency.nil?
   .profile-poll
     .text-center.small
-      GE2019 results for #{other_user.constituency.try(:name) or "Unknown"}
+      Predicted GE2024 results for #{other_user.constituency.try(:name) or "Unknown"}
```

Comment:
Might be worth getting this from `election_year` or similar at some point? Not important though.

Reply:
Right. I was most concerned with the change from previous results to predicted results. But yes, could be tweaked.